scenario/scenario.go

```diff
@@ -484,7 +374,30 @@ func GetTestSignerFn(signerAddress common.Address) func(signer types.Signer, add
 	}
 }
 
-func waitTxnSuccessfullyMined(t *testing.T, ethClient *ethclient.Client, txn *types.Transaction) {
+func GetDummyERC721AssetData(tokenID *big.Int) []byte {
```

Comment:
Why? This is faster.

Reply:
Looking at this again, what I'd really prefer is for us to have something like `assetDataUtils` that handles this for us in a way that is both fast and clear. You're right that `DevUtils` is slow, but there is a pretty easy alternative using the `IAssetData` contract wrapper. 

This is pretty straightforward, so I'm not going to push on this too hard, but I think that having an `assetDataUtils` library would prevent us from making mistakes in these types of functions. Honestly, it would be pretty easy to just put this in a function and have that be part of the library that I'm talking about.

Reply:
I think that's a great idea. Can you open an issue for it?

Reply:
Yes, I’ll do that.